sdk/src/wallet/migration/chrysalis.rs

```diff
@@ -146,9 +144,9 @@ pub(crate) fn migrate_from_chrysalis_data(
     storage_path: &Path,
     // in stronghold the keys are hashed first
     stronghold: bool,
-) -> Result<(Vec<AccountDetailsDto>, Option<String>)> {
+) -> Result<(Vec<AccountDetailsDto>, Option<Value>)> {
     let mut new_accounts: Vec<AccountDetailsDto> = Vec::new();
-    let mut secret_manager_dto: Option<String> = None;
+    let mut secret_manager_dto: Option<Value> = None;
```

Comment:
We could return to_string from this function directly ? Bit weird to return a Value. Or do we need to access its fields ?

Reply:
In L 127 we don't need the extra conversion if we return Value